libr/core/cconfig.c

```diff
@@ -3606,6 +3606,53 @@ R_API int r_core_config_init(RCore *core) {
 	return true;
 }
 
+R_API void radare2_rc(RCore *r) {
```

Comment:
wrong function name. public functions should start with `r_core_`, and i dont think `r_core_radare2_rc` is a good name either